lesson_07/objects/objects_app/src/test/java/com/codedifferently/lesson7/MohamedTest.java

```diff
@@ -0,0 +1,60 @@
+package com.codedifferently.lesson7;
+
+// import java.lang.reflect.Method;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.codedifferently.lesson7.MohamedObjects.Main.CPU;
+import com.codedifferently.lesson7.MohamedObjects.Main.GPU;
+import com.codedifferently.lesson7.MohamedObjects.Main.IllegalYearExeption;
+import com.codedifferently.lesson7.MohamedObjects.Main.cpuBrands;
+import com.codedifferently.lesson7.MohamedObjects.Main.portTypes;
+import java.util.*;
+import org.junit.jupiter.api.Test;
+
+class MohamedTest {
```

Comment:
You have several tests that are really testing the same thing (object construction). Merge those into one constructor test that constructs an instance and asserts that the properties are set correctly. 

Look into the "Arrange-Act-Assert" pattern.